docs/src/udp.rst

```diff
@@ -181,6 +181,23 @@ API
 
     :returns: 0 on success, or an error code < 0 on failure.
 
+.. c:function:: int uv_udp_set_source_membership(uv_udp_t* handle, const char* multicast_addr, const char* interface_addr, const char* source_addr, uv_membership membership)
+
+    Set membership for a source-specific multicast group. Currently implemented only for IPv4.
```

Comment:
The modern network API's don't provide IPv6-specific `mreq_source` structure. Recommended way to implement SSM in IPv6 is [group_source_req](https://msdn.microsoft.com/en-us/library/windows/desktop/bb427441(v=vs.85).aspx) usage, but this may broke back-compatibility in libuv.